src/Option/OptionsQuery.php

```diff
@@ -51,19 +65,20 @@ public static function factory(string|array $props): static
 	 * Returns defaults for the following based on item type:
 	 * [query entry alias, default text query, default value query]
 	 */
-	protected function itemToDefaults(object $item): array
+	protected function itemToDefaults(array|object $item): array
 	{
 		return match (true) {
+			is_array($item),
 			$item instanceof Obj => [
 				'arrayItem',
```

Comment:
@distantnative I don't fully understand the logic, but it looks like with this change there's a mismatch between the query entry alias (still `arrayItem`) and the default queries. Is this intended?

Reply:
It is intended. We are shifting to using `item.` in the queries. You can see in line 165 that we always pass the item as `item` data to the query. However, people have used `arrayItem` and `structureItem` in they blueprints and will continue to do so. So line 73 makes sure that this is passed as alias, retrieved in line 164 and also added as data entry in line 165 to access the item in the query.

Reply:
Makes sense. I was just confused why lines 74-75 and 80-81 were changed but the lines for other item types weren't. Couldn't we change the others to `item` as well? Also a comment in line 163 about the alias (something like "the alias is supported for backwards-compatibility") would be great.

Reply:
I think for the other types we should stick with their naming. I don't think there is any benefit in calling `page` now `item` even yes technically that can be done.

Reply:
You are right, that makes sense. Now I understand the change. 😄